pages/index.tsx

```diff
@@ -1,54 +1,11 @@
 import type { NextPage } from 'next'
-import Head from 'next/head'
-import Image from 'next/image'
 import * as React from "react";
-import Textbox from "../components/textbox";
 import styles from '../styles/Home.module.css'
-import a from '../data/IT2810Høst2018.json'
-import cleanHtmlText from "../functions/helpFunctions";
 
 const Home: NextPage = () => {
-  const content:string =  a.ext_inspera_candidates[0].result.ext_inspera_questions[0].ext_inspera_candidateResponses[0].ext_inspera_response!
   return (
     <div className={styles.container}>
```

Comment:
Trenger headeren å være i en container? 
Kanskje greit å ha til senere uansett

Reply:
Nei, nå ligger den inni bare for å være midtstilt, så det er litt penere og ikke trykt inn øverst til venstre. Men kan fjerne det, for er bare en dummy side akkurat nå